src/main/java/Gronk.java

```diff
@@ -0,0 +1,174 @@
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+import java.util.Arrays;
+import java.util.ArrayList;
+
+import java.io.FileWriter;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+
+public class Gronk {
+    public static String WELCOME = "\tHi, I'm Gronk!\n"
+            + "\tWhat are we up to today?";
+    public static String GOODBYE = "\tSystem closing. Goodbye!";
```

Comment:
Perhaps can leave these variables as ```public static final``` as they should be fixed and not changed at all?

Reply:
Good catch, have fixed this issue.